x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -73,6 +73,7 @@
                                 "cluster:monitor/xpack/watcher/watch/get",
                                 "cluster:admin/xpack/watcher/watch/put",
                                 "cluster:admin/xpack/watcher/watch/delete",
+                                GetLifecycleAction.NAME,
```

Comment:
It's preferable, if possible, to use privilege names, such as `read_ilm` in this case, instead of explicit action names. This offers us the maximum of flexibility, as we don't have to worry about BWC when changing action names.